Mention 'npm install --prefix assets' in more instructions (#4258)
phoenixframework/phoenix: installer/templates/phx_single/README.md

  * Install dependencies with `mix deps.get`<%= if @ecto do %>
  * Create and migrate your database with `mix ecto.setup`<% end %><%= if @webpack do %>

>   * Install Node.js dependencies with `npm install --prefix assets` (on Windows, you may need to do `cd assets && npm install && cd ..` instead)<% end %>

Reviewer: ```suggestion
  * Install Node.js dependencies with `npm install` inside the `assets` directory<% end %>
```
